### ISSUE TITLE
gnuroot wheezy

### ISSUE THREAD

**Issue by pangolin12** [corbinlc/gnuroot#3]
hello!
first I have to say gnuroot its a wonderful app, because I already have a real linux distro installed in my phone (without root). now I can use the shell prompt (for example install packages trough apt-get..., browse through the directories, etc.)

you know, maybe this is weird, but also I would like to use packages like feh (open and view pictures), mplayer (play audio and videos), rtcwake, make a basic script, etc; but then there were some errors (so I am asking for help):

for example:
- I installed feh, but when I try to use it:
# 

root@localhost:/storage/sdcard1/imagen# feh image-00.jpg

feh ERROR: cant open X display. It _is_ running yeah?
# 
- I installed x window, xfce4 but when I execute:
# 

root@localhost:/# startx

X.Org X Server 1.12.4
Release Date: 2012-08-27
X Protocol Version 11, Revision 0
Build Operating System: Linux 3.2.0-4-mv78xx0 armv5tl Debian
Current Operating System: Linux localhost 3.4.0 #1 PREEMPT Wed Mar 27 19:45:12 CST 2013 armv7l
Build Date: 17 December 2013 09:56:27PM
xorg-server 2:1.12.4-6+deb7u2 (Julien Cristau jcristau@debian.org)
Current version of pixman: 0.26.0
Before reporting problems, check http://wiki.x.org to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting, (++) from command line, (!!) notice, (II) informational, (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(==) Log file: "/var/log/Xorg.0.log", Time: Sun Mar 23 20:25:27 2014
(==) Using system config directory "/usr/share/X11/xorg.conf.d"

Fatal server error:
xf86OpenConsole: Cannot Open /dev/tty0 (no such file or directory)

Please consult the X.org foundation support at http://wiki.x.org for help

Please also check the log file at "/var/log/Xorg.0.log" for additional information

Server teminated with error (1). Closing log file.

/usr/bin/xinit: giving up
/usr/bin/xinit: unable to connect to X server: Connection refused
/usr/bin/xinit: server error

root@localhost:/#
# 

so could you please help me how to use correctly this packages with gnuroot?
thanks a lot

**Comment by corbinlc** [comment]
Try the new wheezyx rootfs.  It has xterm installed and launches a vncserver at startup.  Then you can connect with whatever vncviewer you prefer on your android device or from another device/pc on your network.  In the future, it will directly launch an xserver.

**Comment by moocow1452** [comment]
Can this also work for Wheezy x86? I'm having a hard time looking for workarounds for my netbook, but this is still the best app for working with proper linux support on Android.

**Comment by corbinlc** [comment]
Yes, it is possible, you you have to manually install tightvncserver and setup some startup scripts.  I will try to complete an x86 version of the wheezyX root really soon (not much work).  Maybe you can test it when complete?

**Comment by moocow1452** [comment]
Sure, but I got it figured out mostly already between xterm and
tightvncviewer. An Ubuntu x86 would be nice if you had time.

**Comment by amritpati14** [comment]
did anyone have  success while connecting WheezyX with any VNC viewer(e.g. Android VNC viewer) .
Mine is always failing while i try to connect.
Could anyone please write the step by step process and which VN viewer to use ?
Thanks in advance.

@Corbinlc
Mr. Corbin, you are not even replying my email, what kind of useer support i would expect from you????

**Comment by moocow1452** [comment]
Try the real VNC Viewer for Android, apt-get install xterm, and export
USER=root in the terminal before you launch your VNC.
